docker-app/qfieldcloud/core/admin.py

```diff
@@ -918,6 +918,8 @@ class DeltaAdmin(QFieldCloudModelAdmin):
         "content",
         "last_feedback__pre",
         "last_modified_pk",
+        "old_geom",
+        "new_geom",
```

Comment:
the information is basically already shown in the `content` field. But now we might be interested to have an easy look whether it is written also the fields above. Might add some `limit 100 characters` though because it is mainly about having any information at all...